src/isd/batch.py

```diff
@@ -0,0 +1,87 @@
+import gzip
```

Comment:
This allows return types w/o the quote marks below.

```suggestion
from future import __annotations__
import gzip
```